jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/internal/container/ContainerConfigurationHandler.java

```diff
@@ -27,7 +27,6 @@ public ContainerConfigurationHandler() {
   public void initializeMeter(Meter meter) {
     meter
         .upDownCounterBuilder(METRIC_NAME)
-        .ofDoubles()
```

Comment:
@kittylyst Can you check out this PR, especially this line? There was intention of using doubles even though the value is a long

Reply:
@anuraaga Should this value be a long? Is it not popssible to have fractional CPU counts?

Reply:
We use `getLong` below to retrieve so it seems no. Unless we should get double?